src/components/Layout/Layout.test.js

```diff
@@ -13,11 +13,21 @@
 import React from 'react';
 import { expect } from 'chai';
 import { shallow } from 'enzyme';
-import Layout from './Layout';
+import proxyquire from 'proxyquire';
 
 const context = { insertCss: () => {} };
 
-describe('App', () => {
+proxyquire.noCallThru();
+
+const FooterContainer = () => <div>Stubbed footer container.</div>;
+const ConfirmModalContainer = () => <div>Stubbed confirm modal container.</div>;
+
+const Layout = proxyquire('./Layout', {
+  '../Footer/FooterContainer': FooterContainer,
+  '../ConfirmModal/ConfirmModalContainer': ConfirmModalContainer
+}).default;
```

Comment:
I think it's fine that this is here, but in the rest of the Lunch codebase I usually call `proxyquire` before every test. I think it's just so I can modify specifically what it stubs out between every test, though, and you don't need to do that here.